src/styles/blocks/calend.scss

```diff
@@ -0,0 +1,25 @@
+@import "./variables/variables";
+@import "./utils/mixins";
+
+.calendar {
```

Comment:
If you use @import you can add these links only main scss fuk

Reply:
agree